mne/viz/topomap.py

```diff
@@ -236,21 +248,21 @@ def _check_outlines(pos, outlines, head_scale=0.85):
                          .532, .510, .489])
         ear_y = np.array([.0555, .0775, .0783, .0746, .0555, -.0055, -.0932,
                           -.1313, -.1384, -.1199])
-        x, y = pos[:, :2].T
-        x_range = np.abs(x.max() - x.min())
-        y_range = np.abs(y.max() - y.min())
 
         # shift and scale the electrode positions
-        pos[:, 0] = head_scale * ((pos[:, 0] - x.min()) / x_range - 0.5)
-        pos[:, 1] = head_scale * ((pos[:, 1] - y.min()) / y_range - 0.5)
+        if 'center' not in head_pos:
+            head_pos['center'] = 0.5 * (pos.max(axis=0) + pos.min(axis=0))
+        if 'scale' not in head_pos:
+            head_pos['scale'] = 0.85 / (pos.max(axis=0) - pos.min(axis=0))
+        pos -= head_pos['center']
+        pos *= head_pos['scale']
```

Comment:
it comes from @agramfort according to `git blame` :) but that could just be due to refactoring or something. I can add a comment though